### DISCUSSION TITLE
How this Swift PR will impact this project ?

### DISCUSSION THREAD

**Discussion by mackoj** [pointfreeco/swift-case-paths#73] [General]
Hi,

I stumble onto this https://github.com/apple/swift/pull/58940 and I wanted to know how to you think it will impact `swift-case-paths` ?

Thanks,

**Comment by stephencelis** [comment]
@mackoj We'll be watching to see where it goes. For now it appears to be missing the `embed` functionality, so it won't be able to replace Case Paths, but hopefully that functionality gets added soon!

If it does, we'll be able to deprecate this library and depend on the native functionality instead.

If it does not, we might be able to at the very least steal some of the code that was introduced for extraction to simplify what we're currently doing.
